src/tasks/sep/034-op-ink-sep-fusaka-prestate/README.md

```diff
@@ -0,0 +1,21 @@
+# 034-op-ink-sep-fusaka-prestate
+
+Status: [DRAFT, NOT READY TO SIGN]
+
+## Objective
+
+This task uses `op-contract/v4.1.0` OPContractsManager to update the prestate of OP Sepolia and Ink Sepolia to the Fusaka compatible prestate.
+
+## Simulation & Signing
+
+Simulation commands for each safe:
+```bash
+cd src/tasks/sep/034-op-ink-sep-fusaka-prestate
+SIMULATE_WITHOUT_LEDGER=1 just --dotenv-path $(pwd)/.env simulate <council|foundation>
+```
+
+Signing commands for each safe:
+```bash
+cd src/tasks/sep/034-op-ink-sep-fusaka-prestate
+just --dotenv-path $(pwd)/.env sign <council|foundation>
```

Comment:
Let's also add `SKIP_DECODE_AND_PRINT=1` to the just commands, like we did here: https://github.com/ethereum-optimism/superchain-ops/pull/1241/files

The reason for this is because printing and decoding can be very slow, and multiple signers complained about this in the U16A tasks, and the decode and print step is a large reason for the slowness